custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_cover.yaml

```diff
@@ -9,6 +9,7 @@ card_cover:
     ulm_card_cover_icon: false
     ulm_card_cover_color: "blue"
     ulm_card_invert_percent: false
```

Comment:
Not sure your thoughts on breaking changes, but this variable has only been in since v1.2.0 and my PR #1114 - so it's unlikely it's been heavily used yet. Could just include the correct 'ulm_card_cover_invert_percent'? This would clean up some of the logic where we have to check twice below...

Reply:
We cannot be sure how many users have used the variable.
To maintain a streamlined update it's better, in my opinion, to add some legacy support.
At least it will reduce (simple) support questions and issues.

Reply:
Yeah okay agreed :)